corpus/pages/views.py

```diff
@@ -42,5 +46,40 @@ def sig(request, sig_name):
 
     return render(request, "pages/sig.html", args)
 
+
+def team(request):
+    compsoc_members = ExecutiveMember.objects.filter(
+        (Q(core__isnull=True) | Q(core=None))
+        & (Q(user__faculty__isnull=True) | Q(user__faculty=None))
+        & Q(sig__name="CompSoc")
+    )
+    diode_members = ExecutiveMember.objects.filter(
+        (Q(core__isnull=True) | Q(core=None))
+        & (Q(user__faculty__isnull=True) | Q(user__faculty=None))
+        & Q(sig__name="Diode")
+    )
+    piston_members = ExecutiveMember.objects.filter(
+        (Q(core__isnull=True) | Q(core=None))
+        & (Q(user__faculty__isnull=True) | Q(user__faculty=None))
+        & Q(sig__name="Piston")
+    )
+    ieee_core = Core.objects.filter(sig__name="ExeCom").order_by("post")
```

Comment:
We can't add ExeCom as a SIG, it is both wrong logically, and will mess up both current and future code

Reply:
I'm not really sure how else to identify ExeCom members then.
As Executive Members, their SIG will only be one of CompSoc/Diode/Piston.
As Core Members, they can only be part of CompSoc/Diode/Piston/SAC/ExeCom. All these 5 categories have to be displayed separately.

Maybe `sig` isn't the best name for the attribute in the Core model, but this is logically correct.

Reply:
What I have done in the newest changes is include 2 attributes. `is_execom` and `is_sac` which helps us pick out executive committee and sac members without compromising their actual sigs